hook/hook.go

```diff
@@ -376,50 +415,77 @@ func runConfigHook(log log15.Logger, serviceName, hookName string,
 		dockerFilePath = path.Join(repoDir, dockerFilePath)
 	}
 
-	imageName := fmt.Sprintf("%s-%s-%d-%d", serviceName, hookName, hookIndex, hookCounter)
+	imageName := fmt.Sprintf("%s-%s-%d-%d",
+		recv.serviceName, recv.hookName, hookIndex, hookCounter)
 
-	if !buildAndRun(log, imageName, dockerFilePath, runArgs, opts) {
+	if !recv.buildAndRun(log, imageName, dockerFilePath, runArgs) {
 		return
 	}
 
 	success = true
 	return
 }
 
-func buildAndRun(log log15.Logger, imageName, dockerFilePath string,
-	runArgs []string, opts Opts) (success bool) {
+func (recv *regionHookRunner) buildAndRun(log log15.Logger, imageName,
+	dockerFilePath string, runArgs []string) (success bool) {
 
 	log = log.New("Dockerfile", dockerFilePath, "ImageName", imageName)
 
 	log.Debug("buildAndRun() BEGIN")
 	defer log.Debug("buildAndRun() END")
-	log.Info("docker build")
 
+	var logOutput bytes.Buffer
+	var runOutput bytes.Buffer
+
+	defer func() {
+
+		if recv.runOutput != nil {
+			*recv.runOutput <- runOutput
+		}
+
+		logOutput.WriteTo(&recv.logOutput)
+		runOutput.WriteTo(&recv.logOutput)
+	}()
+
+	log.Info("docker build")
 	dockerBuildCmd := exec.Command("docker", "build",
 		"-t", imageName,
 		"-f", dockerFilePath,
 		path.Dir(dockerFilePath),
 	)
-	dockerBuildCmd.Stdout = opts.BuildStdout
-	dockerBuildCmd.Stderr = opts.BuildStderr
+	dockerBuildCmd.Stdout = &logOutput
+	dockerBuildCmd.Stderr = &logOutput
 
 	err := dockerBuildCmd.Run()
 	if err != nil {
-		log.Error("docker build", "Dockerfile", dockerFilePath, "Error", err)
+		log.Error("docker build", "Error", err)
+		log.Info("This is not a problem with porter but with the Dockerfile porter tried to build")
+		log.Info("DO NOT contact Brandon Cook to help debug this issue")
```

Comment:
@phylake — I'm not sure you are getting the point across.  Maybe log with in bright red?  Maybe with ‼️ or two?